beacon/endpoints/rest/response/response_schema.py

```diff
@@ -199,8 +199,8 @@ def build_variant_response(data, qparams):
         yield {
             'variant': variant_func(row),
             'variantAnnotations': variant_annotation_func(row),
-            'variantHandover': None,  # build_variant_handover
-            'datasetAlleleResponses': jsonb(row['dataset_response'])
+            'handovers': None,  # build_variant_handover
+            'datasetAlleleResponses': [jsonb(v) for v in row['dataset_response']]
```

Comment:
`jsonb(list) == list(jsonb)` so what's that change about?

Reply:
This is because I changed the way I aggregate the values (`array_agg` vs `jsonb_agg`). I don't remember the details but I couldn't use `jsonb_agg` any more, so I changed it to the other and, then `jsonb(row)` does not work (I think it has to do with the formatting, although both versions are actually arrays, the formatting is not exactly the same: `array_agg` is shown like `[{}]` and the other one as `{{}}`).